k8s/scripts/core_deployment.sh

```diff
@@ -14,7 +14,7 @@ rm -rf $core_dir
 mkdir -p $core_dir
 cd $core_dir
 
-git clone -b main https://github.com/trustbloc/k8s $core_dir
+git clone -b feat-kms-aws-kms-1 https://github.com/alialkhalidi/k8s $core_dir
```

Comment:
@alialkhalidi is this still here by mistake ?

Reply:
not by mistake, but to until this is merged: https://github.com/trustbloc/k8s/pull/311

Reply:
changes have merged now.